swig/swigResqml2Include.i

```diff
@@ -2218,6 +2236,7 @@ namespace RESQML2_NS
 	public:
 		void setStratigraphicColumnRankInterpretation(StratigraphicColumnRankInterpretation * stratiColumnRankInterp);
 		StratigraphicColumnRankInterpretation * getStratigraphicColumnRankInterpretation() const;
+		std::vector<RESQML2_NS::WellboreMarkerFrameRepresentation *> getWellboreMarkerFrameRepresentationSet() const;
```

Comment:
Should propbably define and use macro for getters.

Reply:
Done.